content/lxc/getting-started.md

```diff
@@ -80,9 +80,11 @@ With that done, the last step is to create an LXC configuration file.
 
 Those values should match those found in /etc/subuid and /etc/subgid, the values above are those expected for the first user on a standard Ubuntu system.
 
-Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use the following systemd command to delegate the cgroup:
-
-    systemd-run --unit=myshell --user --scope -p "Delegate=yes" lxc-start <container-name>
+Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use a systemd command to delegate the cgroup. This part of the documention is missing here and elsewhere:
+- [Ubuntu](https://ubuntu.com/server/docs/containers-lxc)
+- [Debian](https://wiki.debian.org/LXC/CGroupV2#LXC_containers_started_by_non-root)
+- [Arch Linux](https://wiki.archlinux.org/title/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
+- [Fedora](https://fedoraproject.org/wiki/LXC)
```

Comment:
New suggestion after discussion with @brauner :

```suggestion
To run unprivileged containers as an unprivileged user, the user must be allocated an empty delegated cgroup (this is required because of the leaf-node and delegation model of cgroup2, not because of liblxc). See [cgroups: Full cgroup2 support](/lxc/news/2020_03_25_13_03.html#cgroups-full-cgroup2-support) for more information.

Unfortunately, it is not possible to simply start a container from a shell as a user and automatically delegate a cgroup. Therefore, you need to wrap each call to any of the `lxc-*` commands in a `systemd-run` command. For example, to start a container, use the following command instead of just `lxc-start my-container`:

    systemd-run --unit=myshell --user --scope -p "Delegate=yes" lxc-start my-container
```

Reply:
Would it make sense to replace all the following commands by the full commands including the systemd-run part?
It would make it easier to just copy them, but I'm a bit reluctant since it'll distract from the actual commands.

Reply:
Wouldn't it make more sense to have some kind of wrapper script?
Or an alternative?
In the arch wiki I saw this solution:
https://wiki.archlinux.org/title/Cgroups#User_delegation

I don't know enough about the cgroup topic so I would like to know what works, what is secure or more/less secure etc.

Either way I would like to have a way which does not mean to always having to type such a long command (even though everyone could write their own scripts, but thats a bit clunky).
This should be considered for LXD as well, even though that could be implemented in a seperate PR.

Reply:
This sounds like a feature request to me - it's not anything that can be handled in a documentation PR. Want to open an issue?

Reply:
Done lxc/lxd#9783.

Reply:
Ok, so @stgraber closed my Request as it is not related to LXD. Someone can feel free to just copy the information to LXC, as it might be usable there.

Reply:
Also interesting: https://github.com/lxc/linuxcontainers.org/issues/409#issuecomment-1011648365

Reply:
Yes, I think we agree that the current documentation is misleading.
@Dmole , do you want to update your PR with my suggestion or a similar wording? Or should we close this PR and I make one to add the missing information?